src/crdt.rs

```diff
@@ -635,6 +637,7 @@ impl Crdt {
                     v.contact_info.tvu,
                     blob.is_coding()
                 );
+                max_idx = cmp::max(max_idx, blob.get_index().unwrap());
```

Comment:
the blobs should be in order right? can we just look at the last one?

Reply:
or it should just be the same as transmit_index below the errors loop

Reply:
I wan't sure of the in-order thing. If you are certain, I can remove the extra logic.

Reply:
Thanks @sakridge , I think you are right about transmit_index. Changed the code to just use that.

Reply:
Yea, pretty sure. Look at the loop that generates the `orders` vec, it iterates from `transmit_index..received_index` and plucks the things out of the window in order.